src/mono/netcore/sample/wasm/console/Program.cs

```diff
@@ -4,15 +4,9 @@
 using System;
 using System.Threading.Tasks;
 
-public class Test
-{
-    public static async Task<int> Main(string[] args)
-    {
         await Task.Delay(1);
         Console.WriteLine("Hello World!");
         for (int i = 0; i < args.Length; i++) {
             Console.WriteLine($"args[{i}] = {args[i]}");
         }
-        return args.Length;
-    }
-}
\ No newline at end of file
+        return args.Length;
```

Comment:
Was it intended to switch this to a top level program? If so, we should get rid of the indentation.

Reply:
no this was damage reverting